ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/minio_deploy.py

```diff
@@ -103,6 +104,17 @@ def minio_resources(self):
             self.user_form_data["ZDB number"] += 1
             self.number_of_minio_nodes = 2
 
+    @j.baseclasses.chatflow_step(title="Public key")
+    def public_key(self):
+        public_key_file = self.upload_file(
+            """Please add your public ssh key, this will allow you to access the deployed minio container using ssh.
+                Just upload the file with the key. (Optional)"""
+        ).split("\n")
+        if public_key_file:
+            self.public_key = public_key_file[0]
```

Comment:
`self.public_key` now hides the `def public_key` method. Please rename

Reply:
Fix: https://github.com/threefoldtech/jumpscaleX_threebot/pull/745/commits/e2e4f640b4029c7afe0376a96061dd5dc63591b8